tests/e2e/fixtures/react-router-edge-site-v8/app/app.css

```
@@ -0,0 +1,12 @@
+@tailwind base;
```

Stylelint (17.13.0)

[error] Unexpected unknown at-rule "`@tailwind`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)